libcontainer/standard_init_linux.go

```diff
@@ -181,6 +190,25 @@ func (l *linuxStandardInit) Init() error {
 	if err != nil {
 		return err
 	}
+	// Set seccomp as close to execve as possible, so as few syscalls take
+	// place afterward (reducing the amount of syscalls that users need to
+	// enable in their seccomp profiles).
```

Comment:
I'll append the following comment:

> However, this needs to be done before closing the pipe since we need it to pass the seccompFd to the parent.